model/bootstrap/node_info.go

```diff
@@ -25,6 +25,13 @@ const (
 // that is only valid on instances containing private info.
 var ErrMissingPrivateInfo = fmt.Errorf("can not access private information for a public node type")
 
+// NodeMachineAccountPriv contains the private configration need to construct a
+// NodeMachineAccountInfo object. This is used as an intemediary by the bootstrap scripts
+// for storing the private key before generating a NodeMachineAccountInfo.
+type NodeMachineAccountPriv struct {
+	PrivateKey encodable.MachineAccountPrivKey
+}
```

Comment:
Both this file and the `NodeMachineAccountInfo` file are private, so calling it `Priv` doesn't really disambiguate between them. I'd call it `NodeMachineAccountKey` (matches the filename). 
```suggestion
type NodeMachineAccountKey struct {
	PrivateKey encodable.MachineAccountPrivKey
}
```